OJP/OJP_JourneySupport.xsd

```diff
@@ -92,7 +92,17 @@
 					<xs:documentation>Contains the TrainNumber description from NeTEx. If several TrainNumber types exist, use the commercial number.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="TrainName" type="xs:normalizedString" minOccurs="0">
```

Comment:
What is the name of a train ? Journey Pattern name ? Line name ? Train number ? Service Journey name ? or is it a branding attached to the train ?